app/src/pages/inside/settingsPage/settingTabs/demoDataTab/demoDataTab.scss

```diff
@@ -0,0 +1,88 @@
+.demo-data-tab {
+  display: flex;
+  flex-direction: column;
+  width: 80%;
+  max-width: 660px;
+  margin: 0 auto;
+  @media (max-width: $SCREEN_XS_MAX) {
+    width: 100%;
+  }
+}
+
+.description-header {
+  margin-top: 2px;
+  font-size: 12px;
+  font-family: $FONT-SEMIBOLD;
```

Comment:
Please check your styless and rewrite them according rules https://css-tricks.com/poll-results-how-do-you-order-your-css-properties/

Reply:
Fixed